### DISCUSSION TITLE
How to manage static StructureMap files and corresponding Questionnaire and Questionnaire test files

### DISCUSSION THREAD

**Discussion by ekigamba** [opensrp/fhircore#917] [General]
How to manage static StructureMap files and corresponding Questionnaire and Questionnaire test files that are currently located [here](https://github.com/opensrp/fhircore/tree/main/android/eir/src/test/resources/structure-map-questionnaires). Currently we have only the files using StructureMap extraction here from all the modules

The current purpose for keeping them here is:
- To avoid bundling them with the APK
- Easily manually test them
- Store them in version control

**Comment by pld** [comment]
Some context and discussion is in #909

**Reply by pld** [reply]
This is also related to #594, better structuring configurations should help with this and may guide how best to organize this.
